File: app/src/main/java/com/sakurafish/pockettoushituryou/data/local/FoodsData.java
package com.sakurafish.pockettoushituryou.data.local;

import com.google.gson.annotations.SerializedName;
import com.sakurafish.pockettoushituryou.data.db.entity.Foods;
import com.sakurafish.pockettoushituryou.data.db.entity.Kinds;

import java.util.List;

public class FoodsData {
    public static final int KINDS_ALL = 0;

    @SerializedName("data_version")
    private int dataVersion;

    private List<Kinds> kinds;

    private List<Foods> foods;

    public int getDataVersion() {
        return dataVersion;
    }

    public void setDataVersion(int dataVersion) {
        this.dataVersion = dataVersion;
    }

    public List<Kinds> getKinds() {
        return kinds;
    }

    public void setKinds(List<Kinds> kinds) {
        this.kinds = kinds;
    }

    public List<Foods> getFoods() {
        return foods;
    }

    public void setFoods(List<Foods> foods) {
        this.foods = foods;
    }
}

File: app/src/main/java/com/sakurafish/pockettoushituryou/data/local/FoodsData.kt
package com.sakurafish.pockettoushituryou.data.local

import com.sakurafish.pockettoushituryou.data.db.entity.orma.Foods
import com.sakurafish.pockettoushituryou.data.db.entity.orma.Kinds
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Local json data access class with
 * This class has been deprecated from version 3.5.1 due to migration with the Room library
 */
@Deprecated("This class is not used from version 3.5.1. Use 'FoodsAndKinds' class")
@JsonClass(generateAdapter = true)
data class FoodsData(
        @Json(name = "data_version")
        var dataVersion: Int = 0,
        var kinds: List<Kinds>? = null,
        var foods: List<Foods>? = null
)
